src/ORM/Table.php

```diff
@@ -1149,7 +1149,7 @@ public function query()
     /**
      * {@inheritDoc}
      */
-    public function updateAll($fields, $conditions)
+    public function updateAll($fields, $conditions = [])
```

Comment:
This is so dangerous, I'm not sure its a good idea. deleteAll() also requires conditions.